kinto/core/resource/schema.py

```diff
@@ -305,12 +338,43 @@ class JsonPatchBodySchema(colander.SequenceSchema):
     operations = JsonPatchOperationSchema(missing=colander.drop)
 
 
+# Request schemas
+
+
 class RequestSchema(colander.MappingSchema):
-    """Baseline schema for kinto requests."""
+    """Base schema for kinto requests."""
+
+    header = HeaderSchema()
+    querystring = QuerySchema()
+
+
+class GetRequestSchema(RequestSchema):
+    """Base request schema for get requests."""
+
+    querystring = GetQuerySchema()
+
+
+class PatchRequestSchema(RequestSchema):
+    """Base request schema for patch requests."""
+
+    header = PatchHeaderSchema()
+
+
+class CollectionRequestSchema(RequestSchema):
+    """Request schema used with collections."""
+
+    querystring = CollectionQuerySchema()
+
+
+class CollectionGetRequestSchema(CollectionRequestSchema):
+    """Collection get request schema."""
+
+    @colander.instantiate()
+    class querystring(CollectionQuerySchema, GetQuerySchema):
+        pass
```

Comment:
`_fields` is not accepted in plural DELETE, so that's why it's not there. I've tried to omit "General" and just add the Collection or Record according to each type. Turns out Record was not used. :P